in_toto/keylib.go

```diff
@@ -285,6 +285,8 @@ On success it will return nil. The following errors can happen:
   - no valid PKCS8/PKCS1 private key or PKIX public key
   - errors while marshalling
   - unsupported key types
+
+Deprecated: This method has been deprecated.
```

Comment:
I understand having just "This method has been deprecated." for where there's a whole package deprecation note.
But for packages where only specific functions are deprecated so far do we want a longer comment which links to go-witness?

Reply:
I am not sure how far we want to go. At first I thought of putting the deprecation notice only on package level, but then I realized that many IDEs and code editors do not pick up these deprecation notices and they only pick it up for method comments.

Adding a comment to EVERY method seems really toilsome for me, so I tried adding it only to the most important method to make clear that we deprecate this.

We can link to go-witness from there as well, but I kinda want to avoid putting a big notice on every exported variable and method.